flake8 update causes lint failures in master

### Describe the problem
<!-- Describe the problem clearly here. -->
We need to fix 2 new lint errors exposed after an update to flake8

### Source code / logs
<!-- Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem. -->
```
modin/pandas/dataframe.py:201:38: F632 use ==/!= to compare str, bytes, and int literals
modin/pandas/test/test_dataframe.py:1266:22: F632 use ==/!= to compare str, bytes, and int literals
```

--- a/modin/pandas/dataframe.py
+++ b/modin/pandas/dataframe.py
@@ -198,7 +198,7 @@ class DataFrame(object):
             expr: The expression to evaluate. This string cannot contain any
                 Python statements, only Python expressions.
         """
-        if isinstance(expr, str) and expr is "":
+        if isinstance(expr, str) and expr == "":
             raise ValueError("expr cannot be an empty string")
 
         if isinstance(expr, str) and "@" in expr:

--- a/modin/pandas/test/test_dataframe.py
+++ b/modin/pandas/test/test_dataframe.py
@@ -1263,7 +1263,7 @@ def test_cumsum(request, data, axis, skipna):
     # pandas exhibits weird behavior for this case
     # Remove this case when we can pull the error messages from backend
     if name_contains(request.node.name, ["datetime_timedelta_data"]) and (
-        axis == 0 or axis is "rows"
+        axis == 0 or axis == "rows"
     ):
         with pytest.raises(TypeError):
             modin_df.cumsum(axis=axis, skipna=skipna)

